eth/tracers/native/call.go

```diff
@@ -88,13 +88,30 @@ func (f *callFrame) processOutput(output []byte, err error) {
 	}
 }
 
+// stringOpCode is being used to marshal/unmarshal OpCode to/from string which is used by gencodec
+type stringOpCode vm.OpCode
+
+func (c stringOpCode) String() string {
+	return vm.OpCode(c).String()
+}
+
+func (c stringOpCode) MarshalText() ([]byte, error) {
+	return []byte(c.String()), nil
+}
+
+func (c *stringOpCode) UnmarshalJSON(input []byte) error {
+	code := vm.StringToOp(string(input[1 : len(input)-1]))
+	*c = stringOpCode(code)
+	return nil
+}
+
 type callFrameMarshaling struct {
-	TypeString string `json:"type"`
-	Gas        hexutil.Uint64
-	GasUsed    hexutil.Uint64
-	Value      *hexutil.Big
-	Input      hexutil.Bytes
-	Output     hexutil.Bytes
+	Type    stringOpCode `json:"type"`
```

Comment:
I feel like you told me before what's the point of this change but I forgot. Could you please say again?

Reply:
@s1na I am totally fine if you prefer reverting this change